packages/trie/src/trie/trie.ts

```diff
@@ -46,11 +57,29 @@ export class Trie {
     this._root = this.EMPTY_TRIE_ROOT
     this._deleteFromDB = opts?.deleteFromDB ?? false
 
+    if (opts?.db) {
+      this._persistRoot = opts.persistRoot ?? true
+    } else {
+      this._persistRoot = false
+    }
+
     if (opts?.root) {
       this.root = opts.root
     }
   }
 
+  static async make(opts?: TrieOpts) {
```

Comment:
Our pattern for static constructors in the monorepo is to call them create (see `blockchain.create` and `vm.create`). Can you rename this and all the other references to follow that pattern?